ee/ucp/admin/install/install-on-azure.md

```diff
@@ -18,19 +18,20 @@ IP addresses to Kubernetes pods.  The Azure IPAM module requires each Azure
 VM that's part of the Kubernetes cluster to be configured with a pool of
 IP addresses.
 
-You have two options for deploying the VMs for the Kubernetes cluster on Azure:
-- Install the cluster on Azure stand-alone virtual machines. Docker UCP provides
-  an [automated mechanism](#configure-ip-pools-for-azure-stand-alone-vms)
-  to configure and maintain IP pools for stand-alone Azure VMs. 
-- Install the cluster on an Azure virtual machine scale set. Configure the
-  IP pools by using an ARM template like 
-  [this one](#set-up-ip-configurations-on-an-azure-virtual-machine-scale-set).
-
-The steps for setting up IP address management are different in the two
-environments. If you're using a scale set, you set up `ipConfigurations`
-in an ARM template. If you're using stand-alone VMs, you set up IP pools
-for each VM by using a utility container that's configured to run as a
-global Swarm service, which Docker provides.
+You have two options for provisoning IPs for the Kubernetes cluster on Azure:
+- Docker UCP provides an automated mechanism to configure and maintain IP pools 
+  for stand-alone Azure VMs. This service runs within the calico-node daemonset 
+  and by default will provision 128 IP address for each node. This value can be 
```

Comment:
Does this run as part of the calico ds? I can't see that in our calico yaml? Thought it was still just a standalone service (but started by UCP?)

Reply:
Yep, its part of the calico-node DS :)

Reply:
So it, temporarily blindness yesterday :p